tests/templates/kuttl/aas-user-info/10-install-opa.yaml.j2

```diff
@@ -27,7 +27,9 @@ commands:
         name: test-opa
       spec:
         image:
+          repo: docker.stackable.tech/sandbox
```

Comment:
Is this fine to leave in? I'm not quite sure. Any input @sbernauer?

Reply:
This was actually a temporary commit for the integration tests running in jenkins (for the integration test linked in the checklist above). I just forgot to revert it. Thanks for the hint!